kinesis-produce-consume/src/main/java/kinesis/stream/OrdersSource.java

```diff
@@ -0,0 +1,46 @@
+package kinesis.stream;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.integration.support.MessageBuilder;
+import org.springframework.stereotype.Component;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import lombok.extern.java.Log;
+
+/**
+ * 
+ * @author Peter Oates
+ *
+ */
+@Log
+@Component
+public class OrdersSource {
+
+	private OrderProcessor orderOut;
+
+	@Autowired
+	public OrdersSource(OrderProcessor orderOut) {
+		this.orderOut = orderOut;
+	}
+
+	public void sendOrder(Event event) {
+
+		ObjectMapper mapper = new ObjectMapper();
+		String eventString = "";
+		try {
+			eventString = mapper.writeValueAsString(event);
```

Comment:
Can't we rely on the Binder feature to convert payload to the JSON automatically?
Seems for me `spring.cloud.stream.bindings.ordersOut.contentType: application/json` in the config should be enough to meet this requirement. More over I see you use that already there. 
Anything wrong with that approach?